e2e/actions/imageexec.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package actions
+
+import (
+	"bytes"
+	"os/exec"
+	"testing"
+)
+
```

Comment:
I don't like the idea of hard-coding these options that are reflections of the options of the commands. It seems like it would be a pain to keep up to date with. I think just passing in the command and all the arguments / options would be best.